packages/block-library/src/navigation/edit/index.js

```diff
@@ -158,7 +162,7 @@ function Navigation( {
 				isInnerBlockSelected: hasSelectedInnerBlock( clientId, true ),
 			};
 		},
-		[ clientId ]
+		[ clientId, ref ]
```

Comment:
Looks like ref is no longer needed as a dep.
```suggestion
		[ clientId ]
```